src/icons/IconProps.ts

```diff
@@ -0,0 +1,18 @@
+interface IconProps extends Pick<React.SVGProps<SVGSVGElement>, "width"> {
+  size?: "small" | "medium" | "large";
+}
+
+export const getIconSize = (size?: string) => {
```

Comment:
Questa funzione dovrebbe stare in un file separato `get-icon-size.ts` o `icon-utils.ts`.
Le size, a regime, dovranno stare nel tema (ovviamente per il semplice POC va bene così)